### DISCUSSION TITLE
Check my data-centric customer segmentation approach and give feedback.

### DISCUSSION THREAD

**Discussion by shahriar17** [community/community#28540] [Other Feature Feedback, Questions, & Ideas] [answered]
Check out my first project for a DataCamp competition, which is about how to use customer segmentation to improve marketing, customer service, and product development for a company that makes orthopedic equipment.

I'd like every thoughtful person to read the whole report and give me feedback, ideas, and suggestions to help me improve these skills. Please upvote my work if you think my segmentation strategy has helped differentiate customers and my advice can improve the company's overall purpose.

**(removed by moderator)**

Thank you!

**Comment by tuves** [comment, marked as answer]
Hi @shahriar17, we’re glad you’re here 😀

Thanks for sharing your (project/website/etc) with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. 

You had a similar post to this one that was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. We ask that you no longer cross-post.

We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members. For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
